tests/TestCase.php

```diff
@@ -95,4 +96,30 @@ protected function setUpDatabase($app)
             $table->string('password');
         });
     }
+
+    /**
+     * @return string
+     */
+    protected function getAttachmentsTable()
+    {
+        $modelClassName = $this->app['config']->get('nova.froala-field.attachment_model');
+
+        /** @var Model $model */
+        $model = new $modelClassName();
```

Comment:
Extract get table of class string to the own method.

Reply:
That doesn't sound very DRY. Any specific reason for introducing code duplication?